gateway/src/resty/concurrent/timer_task.lua

```diff
@@ -89,6 +89,13 @@ function _M:execute(run_now)
   run_periodic(run_now or false, self.id, self.task, self.args, self.interval)
 end
 
+--- Schedule one task.
+function _M:schedule_one(delay)
+  -- Need to wrap the task in a function that discards the first param (the
+  -- "premature" one sent by ngx.timer.at)
+  ngx.timer.at(delay or 0, function(_, ...) self.task(...) end, unpack(self.args))
```

Comment:
This is going to allocate a function for each call. Is that necessary?

Reply:
Does no longer apply. This method has been replaced.